src/auth/test/auth.controller.spec.ts

```diff
@@ -83,10 +102,57 @@ describe('AuthController', () => {
       //when
       const errors = await validate(changePasswordRequestDto);
       await authController.changePassword(userId, request);
-
       //then
       expect(errors).toHaveLength(0);
       expect(authService.changePassword).toBeCalled();
     });
   });
+
+  describe('Activate account', () => {
+    it('should activate user account', async () => {
+      //given
+      const token = faker.string.sample(64);
+      jest.spyOn(authService, 'activateAccount');
+
+      //when
+      await authController.activateAccount(token);
+
+      //then
+      expect(authService.activateAccount).toHaveBeenCalled();
+    });
+  });
+
+  describe('Reset password', () => {
+    it('should send email with link to reset password', async () => {
+      const request: ResetPasswordEmailRequest = {
+        email: faker.internet.email(),
+      };
+      jest.spyOn(authService, 'generateResetPasswordToken');
+      jest.spyOn(mailService, 'sendResetPasswordEmail');
+
+      await authController.resetPasswordEmail(request);
+
+      expect(authService.generateResetPasswordToken).toHaveBeenCalled();
+      expect(mailService.sendResetPasswordEmail).toHaveBeenCalled();
+    });
+  });
+
+  it('should change password for the provided one via form', async () => {
```

Comment:
I think `should change password` should be enough. The new password is not really provided in the form (as form is on the frontend side) but in a request body